new.go

```diff
@@ -329,6 +341,22 @@ func newBuilder(ctx context.Context, store storage.Store, options BuilderOptions
 	namespaceOptions := defaultNamespaceOptions
 	namespaceOptions.AddOrReplace(options.NamespaceOptions...)
 
+	defaultConfig := config.DefaultConfig()
+	err = defaultConfig.UpdateFromFile(DefaultContainersConfig)
```

Comment:
Can we add this all to config.DefaultConfig("")

Add a option to DefaltConfig to allow user to pass in a DefaultConfig path.  If "" then the code below gets executed.

Reply:
config.New(path string) (*Config, error) is the method to receive filepath.  I update new.go the config.New(path) will be called here. 
 

> Add a option to DefaltConfig to allow user to pass in a DefaultConfig path. If "" then the code below gets executed.
you mean add DefaultConfigpath to ContainersConfig struct(toml table)?